common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -1520,7 +1520,7 @@ def reset_problem(self, _data):
             return {
                 'success': False,
                 # Translators: 'closed' means the problem's due date has passed. You may no longer attempt to solve the problem.
-                'error': _("Problem is closed."),
+                'msg': _("Problem is closed."),
```

Comment:
I'm not positive one can ever see this message (possibly for "survey" questions the button would be enabled after closed date). But if one COULD see it, what should it say?